bodhi/server/metadata.py

```diff
@@ -274,26 +274,26 @@ def insert_updateinfo(self):
         fd, name = tempfile.mkstemp()
         os.write(fd, self.uinfo.xml_dump().encode('utf-8'))
         os.close(fd)
-        self.modifyrepo(name)
+        self.modifyrepo('updateinfo', name)
         os.unlink(name)
 
-    def modifyrepo(self, filename):
+    def modifyrepo(self, filetype, filename):
         """Inject a file into the repodata for each architecture"""
```

Comment:
Let's add an args section to this docblock:

```
Args:
    filetype (basestring): The type of repodata file to be injected. Valid values include 'updateinfo'.
    filename (basestring): The name of the file to inject, such as 'updateinfo.xml'.
```

If you know of more valid options for ```filetype```, please add them to that doc section.